homeassistant/components/atag/__init__.py

```diff
@@ -247,7 +134,7 @@ def available(self):
     @property
     def unique_id(self):
         """Return a unique ID to use for this entity."""
-        return f"{self.coordinator.atag.device}-{self._id}"
+        return f"{self.coordinator.atag.id}-{self._id}"
```

Comment:
Does this change the unique_id?

Reply:
The first part doesn't, that is due to the changed naming in the lib. 
However, the sensors get new values for `self._id`. So the friendly name can be equal to the technical key. There is no reason for the user to use technical keys to reference these entities right? Or do we need to use a mapping?

Reply:
We shouldn't change the unique_id unless there's something wrong with it. Please separate the `self._id` from the entity name. If we do this, this won't be a breaking change.

Reply:
Well I could bring it back in ofcourse, but it would mean additional (non homeassistant specific) code here, which I thought we would want to minimize?

Reply:
We need an entity type for the unique id that's not the same as the entity name.

Reply:
just pushed a small commit that should resolve the break 👍